app/utils/auth.py

```
@@ -1,97 +1,124 @@
+		print(
+			f"[ERROR] | "
+			f"[Payload Validation From Clerk Failure] Reason: Missing required payload fields | "
+			f"Message: Fields from clerk Payload are missing: {ve} | "
+			f"Path: {request_from_context.url.path}"
+		)
+		
+		raise HTTPException(
+			status_code=status.HTTP_401_UNAUTHORIZED,
+			detail=INVALID_BEARER_TOKEN_SCHEMA,
+			headers={"WWW-Authenticate": "Bearer"},
+		)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)